### PR TITLE
Use YAML_SCHEDULE for test 'toolchain_zypper'

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/98391
and https://progress.opensuse.org/issues/68527
Verifications:
http://10.160.64.152/tests/3354 (sles 64bit-smp)
http://10.160.64.152/tests/3359 (sles aarch64)
http://10.160.64.152/tests/3362 (Tumbelweed)
http://10.160.64.152/tests/3365 (Leap 15.4)
https://openqa.opensuse.org/tests/1919108 (Tumbleweed aarch64)
http://openqa.suse.de/tests/7136532 (sles s390x svirt)
http://openqa.suse.de/tests/7136627 (sles ppc64le)
http://openqa.suse.de/tests/7136802 (sles 64bit-smp)

### CODE REVIEW

File: schedule/functional/toolchain_zypper.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: toolchain_zypper
```
Comment (foursixnine) [inline schedule/functional/toolchain_zypper.yaml:2]: please rename this to toolchain instead

Comment (Zaoliang) [reply schedule/functional/toolchain_zypper.yaml]: okay, thanks!

Comment (ggardet) [reply schedule/functional/toolchain_zypper.yaml]: It should be renamed inside the file as well.

File: schedule/functional/toolchain.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    toolchain tests for sles and opensuse
+conditional_schedule:
+    bootloader:
+        BACKEND:
+            'svirt':
+                - installation/bootloader_zkvm
+    kdump_and_crash:
+        DISTRI:
+            sle:
+                - console/kdump_and_crash
```
Comment (ggardet) [inline schedule/functional/toolchain.yaml:14]: Not sure why SLE is different there. IMO, `kdump_and_crash` does not make much sense in a toolchain test.

Comment (Zaoliang) [reply schedule/functional/toolchain.yaml:14]: And I also checked for sle functional on OSD, kdump_and_crash just appears in job 'toolchain_zypper' atm.

File: schedule/functional/toolchain_zypper.yaml (hunk 1 above)
Comment (Zaoliang) [reply schedule/functional/toolchain_zypper.yaml]: sure, thanks!

**Comment by foursixnine** [comment]
@Zaoliang don't forget to make the changes for TW

**Pull request merged**